tools/codegen/gen.py

```diff
@@ -1651,7 +1669,13 @@ def gen_op_headers(g: Union[NativeFunction, NativeFunctionsViewGroup]) -> List[s
         key_fn=key_func,
         env_callable=functionalization_env_callable,
         num_shards=4,
-        sharded_keys={'ops_headers', 'func_definitions', 'func_registrations'}
+        sharded_keys={
+            'ops_headers',
+            'func_definitions',
+            'func_registrations',
+            'func_add_back_views_definitions',
```

Comment:
Do we really need to shard since this is only for view functions?

Reply:
yeah this is definitely overkill. Let me try to unwind it.

Reply:
Ho the out ops are there? Then yes makes sense.